source/tunneling/SecureTunnelingFeature.h

```diff
@@ -114,6 +115,20 @@ namespace Aws
                      */
                     std::string GetEndpoint(const std::string &region);
 
+                    /**
+                     * \brief Get the IotSecureTunneling client
+                     */
+                    virtual std::shared_ptr<AbstractIotSecureTunnelingClient> getClient();
```

Comment:
I guess going with the same reasoning as the feedback posted in the jobs refactoring PR, we rename this to createClient

Reply:
Good call... I actually thought I had done this.